src/components/About/WarningBanner.js

```diff
@@ -0,0 +1,77 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Headline,
+  List,
+  MessageBanner,
+} from '@folio/stripes-components';
+
+import { isVersionCompatible } from '../../discoverServices';
+import css from './About.css';
+
+const WarningBanner = ({
+  interfaces,
+  modules,
+}) => {
+  const modulesArray = _.flatten(_.values(modules));
+  const allInterfaces = modulesArray.reduce((prev, curr) => Object.assign(prev, curr.okapiInterfaces), {});
+
+  const missingModules = Object.keys(allInterfaces).reduce((prev, curr) => {
+    if (!interfaces[curr]) {
+      return [...prev, `${curr} ${allInterfaces[curr]}`];
+    } else {
+      return null;
```

Comment:
does it work right? From my perspective callback in reducer always returns accumulator, and I'n not sure that returning null would work here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Reply:
You are right, it's isn't work in some cases.